Mostly unmatched items, plus a couple fixes I found.

OxyMauron opened the pull request:
<!--
To make reviewing easier, please fill out the table with the IDs.
Detailing changes on the Notes column is appreciated:
E.g:
  Season 2
  Specials (2-5)
  Movie
  TVDB Removed \ Reordered Episodes

TVDB URLs are prefered in the example format (with ID) instead of their address bar redirect (title slug):
   👎 https://thetvdb.com/series/those-obnoxious-aliens/
   👍 https://thetvdb.com/index.php?tab=series&id=75113
-->
Pastel Life and Super Dragon Ball Heroes were found based off my own collection

| AniDB | TVDB/TMDB/IMDB | Notes |
| --- | --- | --- |
| https://anidb.net/anime/2551 | https://thetvdb.com/index.php?tab=series&id=405892 | Season 1 |
| https://anidb.net/anime/2845 | N/A | Other |
| https://anidb.net/anime/3369 | https://thetvdb.com/index.php?tab=series&id=175991 | Season 1 |
| https://anidb.net/anime/4247 | https://thetvdb.com/index.php?tab=series&id=78722  | Seasons 1-2 |
| https://anidb.net/anime/4785 | https://thetvdb.com/index.php?tab=series&id=79108  | Specials. |
| https://anidb.net/anime/6321 | https://thetvdb.com/index.php?tab=series&id=343578 | Season 1 |
| https://anidb.net/anime/6580 | https://www.themoviedb.org/movie/60707 | Movie |
| https://anidb.net/anime/6605 | https://www.themoviedb.org/movie/1087014 https://www.imdb.com/title/tt21141014/ | OVA |
| https://anidb.net/anime/14086 | https://thetvdb.com/index.php?tab=series&id=347734 | Mapping for seasons 2+ |
| https://anidb.net/anime/14049 | https://thetvdb.com/index.php?tab=series&id=349219 | Matched to correct series |

<!-- EXAMPLE ROWS - Enjoy CopyPaste
| https://anidb.net/anime/ | https://thetvdb.com/index.php?tab=series&id= | Season X |
| https://anidb.net/anime/ | https://www.imdb.com/title/tt <br/> https://www.themoviedb.org/movie/ | Movie |
EXAMPLES END -->

BrutuZ commented on anime-list-master.xml:
```diff
@@ -12532,8 +12532,12 @@
-  <anime anidbid="4247" tvdbid="" defaulttvdbseason="" episodeoffset="" tmdbid="" imdbid="">
+  <anime anidbid="4247" tvdbid="78722" defaulttvdbseason="" episodeoffset="" tmdbid="" imdbid="">
     <name>Cubix</name>
+	<mapping-list>
+      <mapping anidbseason="1" tvdbseason="1" start="1" end="13"/>
+      <mapping anidbseason="1" tvdbseason="2" start="14" end="26" offset="-13"/>
+    </mapping-list>
```
Opening is over-indented. The list uses spaces, not tabs
First part is redundant since 1>1 is assumed by default. It doesn't hurt either but adds an extra unnecessary line
```suggestion
    <mapping-list>
      <mapping anidbseason="1" tvdbseason="2" start="14" end="26" offset="-13"/>
    </mapping-list>
```

OxyMauron replied:
Removed tab and unnecessary line.

BrutuZ commented on anime-list-master.xml:
```diff
@@ -13896,8 +13900,11 @@
-  <anime anidbid="4785" tvdbid="" defaulttvdbseason="" episodeoffset="" tmdbid="" imdbid="">
+  <anime anidbid="4785" tvdbid="79108" defaulttvdbseason="0" episodeoffset="3" tmdbid="" imdbid="">
     <name>Ayu Mayu Gekijou</name>
+	<mapping-list>
```
Opening is over-indented. The list uses spaces, not tabs
```suggestion
    <mapping-list>
```

BrutuZ commented on anime-list-master.xml:
```diff
@@ -38627,7 +38634,7 @@
     <name>Mahjong Hishouden: Naki no Ryuu - Hiryuu no Shou</name>
   </anime>
-  <anime anidbid="14049" tvdbid="320002" defaulttvdbseason="0" episodeoffset="1" tmdbid="" imdbid="">
+  <anime anidbid="14049" tvdbid="349219" defaulttvdbseason="1" episodeoffset="0" tmdbid="" imdbid="">
```
No need to specify a 0 offset, better keep it empty for consistency
```suggestion
  <anime anidbid="14049" tvdbid="349219" defaulttvdbseason="1" episodeoffset="" tmdbid="" imdbid="">
```

BrutuZ commented on anime-list-master.xml:
```diff
@@ -38728,6 +38735,14 @@
   </anime>
   <anime anidbid="14086" tvdbid="347734" defaulttvdbseason="1" episodeoffset="" tmdbid="" imdbid="">
     <name>Super Dragon Ball Heroes</name>
+	<mapping-list>
+      <mapping anidbseason="0" tvdbseason="0"/>
+      <mapping anidbseason="1" tvdbseason="1" start="1" end="6"/>
+      <mapping anidbseason="1" tvdbseason="2" start="7" end="19" offset="-6"/>
+      <mapping anidbseason="1" tvdbseason="3" start="20" end="30" offset="-19"/>
+      <mapping anidbseason="1" tvdbseason="4" start="31" end="38" offset="-30"/>
+      <mapping anidbseason="1" tvdbseason="5" start="39" end="48" offset="-38"/>
+    </mapping-list>
   </anime>
```
- Might as well take the chance to change the default to absolute. That will let parser that support it know that the TVDB mapping is correct.
- Opening is over-indented. The list uses spaces, not tabs.
- Season 0>0 is implied

```suggestion
  <anime anidbid="14086" tvdbid="347734" defaulttvdbseason="a" episodeoffset="" tmdbid="" imdbid="">
    <name>Super Dragon Ball Heroes</name>
    <mapping-list>
      <mapping anidbseason="1" tvdbseason="2" start="7" end="19" offset="-6"/>
      <mapping anidbseason="1" tvdbseason="3" start="20" end="30" offset="-19"/>
      <mapping anidbseason="1" tvdbseason="4" start="31" end="38" offset="-30"/>
      <mapping anidbseason="1" tvdbseason="5" start="39" end="48" offset="-38"/>
    </mapping-list>
  </anime>
```

BrutuZ reviewed (changes requested):
This PR has conflicts with the current list, you'll need to address that before it can be merged

OxyMauron commented:
I removed changes to Ayu Mayu Gekijou (AniDB [4785](https://anidb.net/anime/4785)), as this was also done in #377. Tabs have been replaced, and I've removed unnecessary specifications.

BrutuZ reviewed (approved):
LGTM 👍
